### DISCUSSION TITLE
MAUI picker control support for Mac.

### DISCUSSION THREAD

**Discussion by 123JasonWang** [CommunityToolkit/Maui#1836] [General]
When using the MAUI Picker control, Windows works fine, but when running on Mac, the program crashes when clicking on the Picker control. Hopefully the community will fix this.

**Comment by bijington** [comment]
This sounds like a bug in .NET MAUI not something this toolkit can fix. Unless I am missing something?

**Reply by 123JasonWang** [reply]
Currently the picker control is not working on Mac, is there any alternative solution?

**Reply by bijington** [reply]
Have you raised a bug on the .NET MAUI repository? What isn't working?

**Reply by 123JasonWang** [reply]
It seems that maccatalyst does not support Picker controls.

**Reply by bijington** [reply]
Have you raised a bug on the .NET MAUI repository?

**Comment by CliffAgius** [comment]
Closed as not part of the Community Toolkit but the main MAUI repository.
